src/Features/LanguageServer/Protocol/Workspaces/LspWorkspaceManager.cs

```diff
@@ -23,46 +25,32 @@ namespace Microsoft.CodeAnalysis.LanguageServer;
 /// This type is tied to a particular server.
 /// </summary>
 /// <remarks>
-/// This is built to store incremental solutions that we update based on LSP text changes. This solution is <b>eventually consistent</b> with the workspace:
+/// This type provides an LSP view of the registered workspace solutions so that all LSP requests operate
```

Comment:
I recommend ignoring the diff as it is basically non-sensical since this was mostly rewritten